chainer_chemistry/datasets/tox21.py

```diff
@@ -107,20 +107,24 @@ def postprocess_label(label_list):
 
 
 def _get_tox21_filepath(dataset_type):
-    """Returns a filepath in which the tox21 dataset is cached.
+    """Returns a file path in which the tox21 dataset is cached.
 
-    Thie function returns a filepath in which `dataset_type`
+    This function returns a file path in which `dataset_type`
     of the tox21 dataset is cached.
-    Not that this function does not check if the dataset actually
-    has been downloaded or not.
+    Note that this function does not check if the dataset has actually
+    been downloaded or not.
 
     Args:
         dataset_type(str): Name of the target dataset type.
             Either 'train', 'val', or 'test'.
 
-    Returns (str): filepath for the tox21 dataset
+    Returns (str): file path for the tox21 dataset
 
     """
+    if dataset_type not in _config.keys():
+        raise ValueError("Invalid dataset type '{}'. Accepted values are "
+                         "'train', 'val' or 'test'.".format(dataset_type))
```

Comment:
Adding this check means that we will add a spec to raise appropriate error when dataset_type is wrong.
To check this, I appreciate if you can implement test to catch this valueerror when wrong dataset_type is passed.

Test is implemented at here.
https://github.com/pfnet-research/chainer-chemistry/blob/master/tests/datasets_tests/test_tox21.py

implementation will be like this
https://github.com/pfnet-research/chainer-chemistry/blob/81b18c6d1b281dbe66673b7d9a8117aff417af77/tests/dataset_tests/preprocessors_tests/test_ggnn_preprocessor.py#L32-L33